.github/ci_commit_pins/xla.txt

```diff
@@ -1 +1 @@
-b3342319e96a0becd139019620d8665605b78475
+1693c1c728c05a3afda946415acc9dfd6adf421d
```

Comment:
Why does this need an xla side change?

Reply:
XLA uses SymIntNode, so it needs to track the change